### DISCUSSION TITLE
webpack5 build vue workflow using tailwindcss, did't work properly

### DISCUSSION THREAD

**Discussion by dealdot** [tailwindlabs/tailwindcss#8672] [Help]
Hello, recently I made the vue developing workflow from scratch with webpack5, add features using tailwindcss, my configuration about postcss as follows:

```{
            loader: "postcss-loader",
            options: {
              postcssOptions: {
                plugins: [
                  ["tailwindcss"],
                  [
                    "autoprefixer",
                    {
                      overrideBrowserslist: ["last 4 version"],
                    },
                  ],
                ],
              },
            },
          },
```

But I got the error :  Module not found: Error: Can't resolve '/img/grid.svg' , what is the problem please?

**Comment by rohmann** [comment]
Hi @dealdot,

Hard to tell without seeing the repo, but it sounds simply like that file doesn't exist, or at least not at the specified location. It could also be that you need to import from a relative path like `./img/grid.svg` (note the dot).
